### ISSUE TITLE
Version 3.8 bugs

### ISSUE THREAD

**Issue by world33** [strohne/Facepager#56]
Since updating to the 3.8 version I get a couple of errors/bugs.
1) When clicking on the presets I get this error:

---------------------------
Facepager
---------------------------
Error loading preset:No JSON object could be decoded
---------------------------
OK   
---------------------------

It also happens when deleting a preset template

2) When I try to delete multiple selected nodes it deletes only one at the time or it crashes with the following error:

```
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	Facepager.exe
  Application Version:	0.0.0.0
  Application Timestamp:	549debdc
  Fault Module Name:	python27.dll
  Fault Module Version:	2.7.6150.1013
  Fault Module Timestamp:	527fcf52
  Exception Code:	c0000005
  Exception Offset:	000aad72
  OS Version:	6.1.7601.2.1.0.256.48
  Locale ID:	1033
  Additional Information 1:	0a9e
  Additional Information 2:	0a9e372d3b4ad19135b953a78882e789
  Additional Information 3:	0a9e
  Additional Information 4:	0a9e372d3b4ad19135b953a78882e789

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt
```

**Comment by world33** [comment]
With version 3.7 the first error still occurs while the second one does not and it is possible to delete multiple nodes with no issue.

**Comment by strohne** [comment]
Thanks for the bug report...this is due to a failure in a new preset. I will fix it in a few days. Happy holidays.

**Comment by strohne** [comment]
I cannot reproduce these errors, presets shipping with the installer look ok.

1. Maybe you have manually changed one of the presets in the preset folder? 

2. Maybe the crashing occurs on Windows 7 systems only? Do you have another machine available to test version 3.8?

**Comment by world33** [comment]
Hello,

Thank you for your reply.
For the first error I had to delete the following custom preset under my Windows user name Facepager folder.

``
{
  "columns": [
    "id",
    "link",
    "category",
    "about",
    "company_overview",
    "description",
    "general_info",
    "impressum",
    "mission",
    "affiliation",
    "contact_address",
    "location",
    "phone",
    "emails",
    "website",
    "founded",
    "featured_video",
    "start_info",
    "can_post",
    "verification_status",
    "app_links",
    "press_contact",
    "picture",
      ],
  "options": {
    "params": {
      "<page>": "<Object ID>"
    },
    "relation": "<page>?fields=id,link,category,about,company_overview,description,general_info,impressum,mission,affiliation,contact_address,location,phone,emails,website,founded,featured_video,start_info,can_post,verification_status,app_links,press_contact,picture",
    "pages": 1
  },
  "name": "Get Page Info",
  "module": "Facebook",
  "description": "This preset retrieves the following items (if available and published):\n\nid\nlink\ncategory\nabout\ncompany_overview\ndescription\ngeneral_info\nimpressum\nmission\naffiliation\ncontact_address\nlocation\nphone\nemails\nwebsite\nfounded\nfeatured_video\nstart_info\ncan_post\nverification_status\napp_links\npress_contact\npicture"
}
``

There must be something wrong with the json code not big deal.

In regards to the second error I am not sure what it caused it I just know that 3.7 version does not reproduces the bug or crash so I will stick to this version for the moment.

Happy holiday!

**Comment by strohne** [comment]
There is one wrong comma after  `"picture",`, that's all.
I will dig into the second error another day.

**Comment by strohne** [comment]
I think the second error is fixed now, pls try the latest release (3.8.1).

**Comment by world33** [comment]
Thanks I will give it a try. I open a new ticket for the last issue I encountered.
 Cheers.
